README.md

```diff
@@ -110,8 +109,8 @@ in your `WORKSPACE` file (or import from a `.bzl` file:
 ```
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories")
 
-KOTLIN_VERSION = "1.3.31"
-KOTLINC_RELEASE_SHA = "107325d56315af4f59ff28db6837d03c2660088e3efeb7d4e41f3e01bb848d6a"
+KOTLIN_VERSION = "1.3.50"
```

Comment:
Note: Necessary to properly illustrate due to #210 - prior to ***1.3.40*** there is a different enforcement mechanism (a sloppy string comparison) which over-permits, concealing some issues.